_posts/2024-10-30-lower-your-cost-on-opensearch-using-binary-vectors.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title:  "Optimize your OpenSearch costs using binary vectors"
+authors:
+- heemin
+- junqiu
+- vamshin
+- dylantong
+- kolchfa
+date: 2024-10-30 00:00:00 -0700
```

Comment:
Please update the date to 2024-11-21 and we'll publish this on Thursday,

Reply:
Updated